runtime/colors/README.txt

```diff
@@ -54,36 +59,80 @@ autocommand event.  For example:
 	augroup END
 
 To customize a colorscheme use another name, e.g.  "~/.vim/colors/mine.vim",
-and use `:runtime` to load the original colorscheme:
+and use ":runtime" to load the original colorscheme:
+
 	" load the "evening" colorscheme
 	runtime colors/evening.vim
 	" change the color of statements
 	hi Statement ctermfg=Blue guifg=Blue
 
-To see which highlight group is used where, find the help for
-"highlight-groups" and "group-name".
+To see which highlight group is used where, see ":help highlight-groups" and
+":help group-name".
 
 You can use ":highlight" to find out the current colors.  Exception: the
 ctermfg and ctermbg values are numbers, which are only valid for the current
-terminal.  Use the color names instead.  See ":help cterm-colors".
+terminal.  Use the color names instead for better portability.  See
+":help cterm-colors".
 
-The default color settings can be found in the source file src/highlight.c.
+The default color settings can be found in the source file src/syntax.c.
```

Comment:
This change looks wrong. In Vim it's in `src/highlight.c` and in Nvim it's in `src/nvim/highlight_group.c`.

Reply:
Might as well take the opportunity to correct that.